pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAlternativeTitles.scala

```diff
@@ -15,6 +16,9 @@ import uk.ac.wellcome.sierra_adapter.model.SierraBibNumber
 //
 // 246 is only used when indicator2 is not equal to 6, as this is used for
 // the work:lettering field
+//
+// If there exists a $5 subfield with contents `UkLW`, this is Wellcome
+// Library specific and should be excluded.
```

Comment:
Be more specific about "this" – when I first read this, I wasn’t clear if it meant "the individual subfield" or "the entire record".

```suggestion
// Any $5 subfield with contents `UkLW` is Wellcome Library-specific and
// should be omitted.
```